src/menuItems/MenuItem.js

```diff
@@ -0,0 +1,122 @@
+import crel from 'crelt';
+
+const SVG = 'http://www.w3.org/2000/svg';
+const XLINK = 'http://www.w3.org/1999/xlink';
+
+const prefix = 'ProseMirror-icon';
+
+function hashPath(path) {
+  let hash = 0;
+  for (let i = 0; i < path.length; i++)
+    hash = ((hash << 5) - hash + path.charCodeAt(i)) | 0;
+  return hash;
+}
+
+// Work around classList.toggle being broken in IE11
+function setClass(dom, cls, on) {
+  if (on) dom.classList.add(cls);
+  else dom.classList.remove(cls);
+}
+
+function translate(view, text) {
+  return view._props.translate ? view._props.translate(text) : text;
+}
+
+function buildSVG(name, data) {
+  let collection = document.getElementById(prefix + '-collection');
+  if (!collection) {
+    collection = document.createElementNS(SVG, 'svg');
+    collection.id = prefix + '-collection';
+    collection.style.display = 'none';
+    document.body.insertBefore(collection, document.body.firstChild);
+  }
+  const sym = document.createElementNS(SVG, 'symbol');
+  sym.id = name;
+  sym.setAttribute('viewBox', '0 0 ' + data.width + ' ' + data.height);
+  const path = sym.appendChild(document.createElementNS(SVG, 'path'));
+  path.setAttribute('d', data.path);
+  collection.appendChild(sym);
+}
+
+export function getIcon(icon) {
```

Comment:
document how this works? do we want to use our own svgs?

Reply:
we added some copied code from prosemirror core libraries so we can make minor changes the library is not supporting.
this function basically build an icon element for menu items according to a spec passed to her. were not really using it, just needed it as dependency for another function

Reply:
got it